ReactiveCocoa/Objective-C/NSControl+RACCommandSupport.h

```diff
@@ -8,7 +8,7 @@
 
 #import <Cocoa/Cocoa.h>
 
-@class RACCommand;
+#import "RACCommand.h"
```

Comment:
It seems that we can write as `@class RACCommand<__contravariant InputType>;` here.